index.html

```diff
@@ -27,21 +27,163 @@ <h4 class="dev">I’m a Software Developer</h4>
+      </p>
+      <ul class="categories">
+        <li>css</li>
+        <li>html</li>
+        <li>bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button type="button" class="btn">See Project</button>
+    </section>
+    <section class="projects">
+      <div class="project">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+      <div class="project">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+      <div class="project">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+      <div class="project">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+      <div class="project">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+      <div class="project">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+    </section>
+    <section class="abouts">
+      <div class="about">
```

Comment:
[OPTIONAL]

- [ ] This div here isn't necessary since it's the only child of the section, you can remove it if you want. 😁